lib/board.rb

```diff
@@ -2,32 +2,58 @@ class Board
   attr_reader :grid
 
   def initialize
-    @grid = top_row
+    @grid = numbered_row
     build_grid
-    add_numbers
   end
 
   def to_s
-    grid.to_s
+    grid
   end
 
   private
 
-  def row
-    "|  |  |  |\n|__|__|__|\n".freeze
+  def numbered_row
+    "   1  2  3\n".freeze
+  end
+
+  def build_grid
+    add_top_row
+    add_rows
+  end
+
+  def add_top_row
+    @grid += top_row
   end
 
   def top_row
-    " __ __ __\n".freeze
+    "   __ __ __\n".freeze
   end
 
-  def build_grid
-    3.times do
-      @grid += row
-    end
+  def add_rows
+    @grid += rows
   end
 
-  def add_numbers
+  def rows
+    first_labeled_row + second_labeled_row + third_labeled_row
```

Comment:
This does read better than the previous version with the array indexing inlined.

Reply:
✨